nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -167,6 +171,8 @@ def get_weight_tensor_port_ids(node: NNCFNode, graph: NNCFGraph) -> List[Optiona
 
     @staticmethod
     def get_weight_name(nncf_graph: NNCFGraph, target_point: PTTargetPoint) -> str:
+        if is_experimental_torch_tracing_enabled():
+            return target_point.target_node_name
```

Comment:
I would suggest setting layer_name to be the same as target_node_name in graph nodes to avoid adding this 'if' in all algorithms.